packages/cli/package.json

```diff
@@ -21,13 +21,15 @@
     "node": ">=8.0.0"
   },
   "scripts": {
-    "build": "rm -rf lib && tsc -b",
+    "build": "rm -rf lib && yarn run prepack",
     "docs": "yarn oclif-dev readme --multi --dir=../docs/command-line-interface && yarn prettier ../docs/command-line-interface/*.md --write",
     "lint": "tslint -c tslint.json --project tsconfig.json",
+    "prepack": "tsc -b && oclif-dev manifest && oclif-dev readme",
```

Comment:
remove.
and make `prepublishOnly: "yarn build && oclif-dev-manifed"`

Reply:
I was reviving the old codebase. Removed.